contracts/common/misc/Drainable.sol

```diff
@@ -0,0 +1,33 @@
+pragma solidity ^0.5.2;
+
+import {DepositManagerStorage} from "../../root/depositManager/DepositManagerStorage.sol";
+
+import {IERC20} from "openzeppelin-solidity/contracts/token/ERC20/IERC20.sol";
+import {IERC721} from "openzeppelin-solidity/contracts/token/ERC721/IERC721.sol";
+
+
+contract Drainable is DepositManagerStorage {
+
+  function drainTokens(
+    address[] calldata tokens,
+    uint256[] calldata values,
+    address destination
+  ) external onlyGovernance {
+    for (uint256 i = 0; i < tokens.length; i++) {
+      if (registry.isERC721(tokens[i])) {
```

Comment:
Let's remove this. Someone could send unmapped tokens to this contract as well. Have separate functions for draining 20/721s.